Services/data/devicemodels/scripts/EmergencyValveRelease-method.js

```diff
@@ -19,7 +19,7 @@
 /*jslint unparam: true*/
 function main(context, previousState, previousProperties) {
 
-    log("Starting 'Emergency Valve Release' method simulation");
+    log("Starting 'EmergencyValveRelease' method simulation.");
```

Comment:
I noticed the IncreasePressure-method.js file and a few others has an addition of "Javascript" in the log message. For consistency you could either take it our there or add it in here as well.